apps/api/v2/src/modules/organizations/routing-forms/controllers/organizations-routing-forms-responses.controller.ts

```diff
@@ -73,6 +75,7 @@ export class OrganizationsRoutingFormsResponsesController {
 
   @Post("/")
   @ApiOperation({ summary: "Create routing form response and get available slots" })
+  @UseGuards(OrAuth([RolesGuard, IsUserRoutingForm]))
```

Comment:
How does `IsUserRoutingForm` work here? It extracts routing form id as follows:
```
const routingFormId: string = request.params.routingFormId;
```

but there is no param `routingFormId` when making a post request.

Reply:
The full request has the param right e.g. it gets it from there.
{{API_URL}}/v2/organizations/7/routing-forms/69e1694a-d0d5-418a-b675-ff899298a3a3/responses?start=2025-06-30T18:30:00.000Z&end=2025-07-23T09:00:00Z&queueResponse=true